File: tests_wraps/src/main/java/com/github/klee0kai/test/car/model/Bumper.java
package com.github.klee0kai.test.car.model;

import java.util.UUID;

public class Bumper {

    public static int createCount = 0;
    public String uuid = UUID.randomUUID().toString();
    public String qualifier = null;

    public Bumper() {
        createCount++;
    }

}

File: tests_wraps_kotlin/src/main/kotlin/com/github/klee0kai/test/car/model/Bumper.kt
package com.github.klee0kai.test.car.model

import java.util.*

class Bumper {
    @JvmField
    var uuid = UUID.randomUUID().toString()

    @JvmField
    var qualifier: String? = null

    init {
        createCount++
    }

    companion object {
        @JvmField
        var createCount = 0
    }
}
